scaper/core.py

```diff
@@ -1794,45 +1808,34 @@ def _generate_audio(self, audio_path, ann, reverb=None,
                             tmpfiles_internal.append(
                                 tempfile.NamedTemporaryFile(
                                     suffix='.wav', delete=False))
+                            
                             # synthesize edited foreground sound event
-                            tfm.build(e.value['source_file'],
-                                      tmpfiles_internal[-1].name)
-                            # if time stretched get actual new duration
-                            if e.value['time_stretch'] is not None:
-                                fg_stretched_duration = soundfile.info(
-                                    tmpfiles_internal[-1].name).duration
-
+                            event_audio_array, event_audio_rate = soundfile.read(
```

Comment:
inconsistent with earlier `event_sample_rate`. Let's use `event_audio` and `event_sr` everywhere.